docs/guides/medical.md

```diff
@@ -0,0 +1,181 @@
+
+![Triaż w menu](../_data/guides/medical/triange_menu.png)
+
+![Triaż - przydzielanie](../_data/guides/medical/triange.png)
+
+## Krew, osocze, sól fizjologiczna
+
+W Medicalu każdy żołnierz ma 6 litrów krwi z organizmie. Każdy próg ma przypisany przedział ilości krwi występującej w danej chwili w organizmie żołnierza. Występują cztery progi:
+- Stracił trochę krwi (Lost some blood) (6,0 – 5,1)
+- Stracił dużo krwi (Lost a lot of blood) (5,1 – 4,2)
+- Stracił bardzo dużo krwi (Lost a large amount of blood) (4,2 – 3,6)
+- Stracił krytyczną ilość krwi (Lost a fatal amount of blood) (3,6 – 3,0)
+
+Te wartości jako medyk musisz zapamiętać. Podczas rozgrywki musisz oszacować ile potrzebujesz krwi, aby ranny/ranni pod twoją opieką odzyskali zdolność bojową.
+
+Wielokrotnie podczas rozgrywek doświadczysz jako medyk sytuacji, w której ilość krwi dostępnej do przetoczenia będzie mocno ograniczona lub nawet krytycznych sytuacjach nie wystarczająca do ustabilizowania wszystkich rannych. Oczywiście zdarzają się również sytuację, kiedy zapasów krwi jest bardzo dużo oraz jest możliwość szybkiego i bezpiecznego dostarczenia zapasów. Warto wtedy skorzystać z takiej możliwości i wlać krwi do całkowitego uzupełnienia braków. Minimalizuje to ryzyko utraty przytomności
+
+Przyjmuje się, że w razie uszczuplonych zapasów krew podaje się tylko nieprzytomnym lub osobom ze znaczną utratą krwi, a tym samym granicznym ciśnieniem krwi w aspekcie przytomności. Każdy członek drużyny powinien w miarę możliwość posiadać przy sobie 0,5 - 1 l krwi, co nie tylko odciąży medyka, ale również pozwoli medykowi korzystać z zapasów rannego. 
+
+W kwestii przetaczania krwi można sugerować się poniższą rozpiską, ale nie jest to sztywna zasada:
+- Stracił trochę krwi - nie podajemy krwi, ponieważ w zdecydowanej większości przypadków ciśnienie krwi oraz tętno są w granicach normy 
+- Stracił dużo krwi - podajemy 0,5 l
+- Stracił bardzo dużo krwi – podajemy 1 l
+- Stracił krytyczną ilość krwi – podajemy 1,5 l
+
+## Ciśnienie krwi i tętno
+Ciśnienie oraz tętno są właściwie najważniejszymi czynnikami w Medicalu. To one są wskaźnikiem w jakim stanie jest organizm żołnierza. Rolą medyka jest rozpoznanie za pomocą pomiaru ciśnienia oraz tętna jakiej należy udzielić pomocy rannemu. Wpływ na ciśnienie oraz tętno ma mnóstwo czynników, m.in. skala utraty krwi, morfina, adrenalina, zmęczenie, ból itp. 
+
+![Tabela ciśnień](../_data/guides/medical/pressure.png)
+
+Dla tętna przyjmuje się następujące wartości:
+- Niskie – poniżej 60
+- Normalne – 60 – 100
+- Wysokie – powyżej 100
+	
+	- Niskie ciśnienie, tętno - Przetoczyć krew, podać adrenalinę. Jeżeli doszło do przedawkowania morfiny, podać tą samą liczbę dawek adrenaliny i odczekać.
+	- Wysokie ciśnienie, tętno – Podać morfinę lub adenozynę
+	- Niskie ciśnienie, wysokie tętno – Przetoczyć krew. 
+	
```

Comment:
What happened here? I don't think this is a sublist but rather second list. Please fix that as this is not fine:
![obraz](https://user-images.githubusercontent.com/30000580/109433028-7a01c800-7a0e-11eb-937f-57c4a138d1e8.png)